filebeat/input/redis/harvester.go

```diff
@@ -146,6 +155,7 @@ func (h *Harvester) Run() error {
 			"duration": mapstr.M{
 				"us": log.duration,
 			},
+			"role": role,
```

Comment:
Can you add an assertion [here](https://github.com/elastic/beats/blob/755831e/filebeat/tests/system/test_redis.py#L49) as well? Just so we get some coverage for this. Thanks.

`assert "redis.slowlog.role" in output`

Reply:
@mauri870 thanks! I've added that now, I don't see the test running in the checks though.

Reply:
I talked with the team and confirmed that it indeed runs in integration tests, but only after the PR is approved.